subprojects/core/src/integTest/groovy/org/gradle/NativeServicesIntegrationTest.groovy

```diff
@@ -109,7 +109,7 @@ class NativeServicesIntegrationTest extends AbstractIntegrationSpec {
             import org.gradle.internal.nativeintegration.NativeCapabilities
 
             tasks.register("doWork", WorkerTask)
-            println("Uses native integration in daemon: " + NativeServices.instance.createNativeCapabilities().useNativeIntegrations())
+            println("Uses native integration in daemon: " + NativeServices.INSTANCE.createNativeCapabilities().useNativeIntegrations())
```

Comment:
💭  I am not a fan of this change, but this logic in the test should probably be rewritten to not rely on the way `NativeServices` declares its services.